docs/sphinx/_toc.yml.in

```diff
@@ -31,6 +31,8 @@ subtrees:
   - file: how-to/logging
   - file: how-to/unified_memory
     title: Unified Memory
+  - file: how-to/virtual_memory
+    title: Virtual Memory
```

Comment:
```suggestion
    title: HIP Virtual Memory
```

Reply:
@lpaoletti We do not add the HIP at how-to titles only at references. Do we have some rules in the documentation guideline about this?